CRFModuleValidation/Cardio_12MT.json

```diff
@@ -21,33 +40,42 @@
                            "identifier"   : "wearingFitbit",
                            "type"         : "instruction",
                            "title"        : "Wearing your fitbit?",
-                           "imageBefore"  : "fitbitCheckIcon",
                            "text"         : "Before continuing on with the 12 minute test, please make sure you have your fitbit on your wrist.",
-                           "actions"           : { "goForward": { "buttonTitle": "It’s on" }}
+                           "image"        : { "imageName": "fitbitCheckIcon", "placementType": "topBackground" },
+                           "colorTheme"   : { "backgroundColor" : "perrywinkle", "usesLightStyle" : true },
+                           "actions"      : { "goForward": { "buttonTitle": "It’s on" }},
+                           "viewTheme"    : { "viewIdentifier": "Instruction", "storyboardIdentifier": "ActiveTaskSteps" }
                            },
                            {
                            "identifier"   : "volumeUp",
                            "type"         : "instruction",
                            "title"        : "Volume turned up?",
-                           "imageBefore"  : "volumeCheckIcon",
                            "text"         : "Bring your phone with you and turn up your phone volume so you can hear the instructions while you are moving.",
-                           "actions"           : { "goForward": { "buttonTitle": "It’s turned on" }}
+                           "image"        : { "imageName": "volumeCheckIcon", "placementType": "topBackground" },
+                           "colorTheme"   : { "backgroundColor" : "perrywinkle", "usesLightStyle" : true },
+                           "actions"      : { "goForward": { "buttonTitle": "It’s turned on" }},
+                           "viewTheme"    : { "viewIdentifier": "Instruction", "storyboardIdentifier": "ActiveTaskSteps" }
                            },
                            {
                            "identifier"   : "goOutside",
                            "type"         : "instruction",
                            "title"        : "Go outside",
-                           "imageBefore"  : "goOutsideIcon",
                            "text"         : "Start by going oustide and standing still to measure your resting heart rate before you start moving.",
```

Comment:
Seems like we should be having them stand still for a certain amount of time before they even start taking their resting heart rate. Just walking outside could bump it up significantly, and what if they had to climb stairs or go up a hill to get out to where they're starting their run? (Also shouldn't a resting heart rate generally be taken sitting down?)

These are probably questions better posed to the study designers, but they're not making this pull request, you are... ;)

Reply:
Yup. I've asked. And I've pointed out why I was gathering location data from the start of the task, but they don't want it. *shrug* I also wanted to put the volume check after the go outside step to give the user a bit more time for their heart rate to come to resting.